npc/re/warps/cities/izlude.txt

```diff
@@ -106,7 +106,7 @@ OnTouch:
 		erasequest(21001);
 	if (isbegin_quest(21002))
 		erasequest(21002);
-	warp "izlude" + strnpcinfo(2),196,209;
+	warp "izlude",196,209;
```

Comment:
Are you sure about that? @Lemongrass3110 told me that player from `int_land01` are warped to `izlude_a`, `int_land02` are warped to `izlude_b` etc.. which should be the case before this commit
```
int_land01,49,57,0	duplicate(noviship#izlude)	noviship#_a	WARPNPC,2,2
int_land02,49,57,0	duplicate(noviship#izlude)	noviship#_b	WARPNPC,2,2
int_land03,49,57,0	duplicate(noviship#izlude)	noviship#_c	WARPNPC,2,2
int_land04,49,57,0	duplicate(noviship#izlude)	noviship#_d	WARPNPC,2,2
```